opm/parser/eclipse/EclipseState/Schedule/MSW/SegmentSet.cpp

```diff
@@ -21,6 +21,11 @@
 #include <cmath>
 #include <map>
 
+#ifdef _WIN32
+#define _USE_MATH_DEFINES 
+#include <math.h>
```

Comment:
I don't like this solution. We already include cmath (so math.h is pointless), and this is just the define.

It should either be moved to be a build parameter, `_USE_MATH_DEFINES` should be above `#include <cmath>` or we should define pi ourselves as a `constexpr`.